src/shared/telemetries/telemetrysender/config.go

```diff
@@ -6,17 +6,19 @@ import (
 )
 
 const (
-	TelemetryAPIAddressKey       = "telemetry-address"
-	TimeoutKey                   = "telemetry-client-timeout"
-	CloudClientTimeoutDefault    = "30s"
-	TelemetryEnabledKey          = "telemetry-enabled"
-	TelemetryEnabledDefault      = false
-	TelemetryMaxBatchSizeKey     = "telemetry-max-batch-size"
-	TelemetryMaxBatchSizeDefault = 100
-	TelemetryIntervalKey         = "telemetry-interval-seconds"
-	TelemetryIntervalDefault     = 5
-	TelemetryAddressDefault      = "https://app.otterize.com/api/telemetry/query"
-	EnvPrefix                    = "OTTERIZE"
+	TelemetryAPIAddressKey                = "telemetry-address"
+	TimeoutKey                            = "telemetry-client-timeout"
+	CloudClientTimeoutDefault             = "30s"
+	TelemetryEnabledKey                   = "telemetry-enabled"
+	TelemetryEnabledDefault               = false
+	TelemetryMaxBatchSizeKey              = "telemetry-max-batch-size"
+	TelemetryMaxBatchSizeDefault          = 100
+	TelemetryIntervalKey                  = "telemetry-interval-seconds"
+	TelemetryIntervalDefault              = 5
+	TelemetryAddressDefault               = "https://app.otterize.com/api/telemetry/query"
+	TelemetrySnapshotResetIntervalKey     = "telemetry-snapshot-reset-interval-duration"
```

Comment:
I think the name "Snapshot telemetry" might be a little bit intimidating.

Reply:
Fixed it to reset which is actually what it is